### PR TITLE
Mobile Menu feature

### PR DESCRIPTION
**Pull request by gak92**
In this pull request, I worked on the mobile menu which is instructed here: https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md

I added:
- Add the close button
- Add functionality when clicking on the hamburger to open the mobile menu
- Add functionality when clicking on the close button to close the mobile menu
- Add functionality when clicking on any icon then move to the corresponding section and close the mobile menu

### CODE REVIEW

**Review by Mre55 — COMMENTED** [review]
**Hi @gak92,**

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :negative_squared_cross_mark:

You have submitted a project, that linter check is not passing. Kindly fix that.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Comment by gak92** [comment]
> **Hi @gak92,**
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request ❎
> You have submitted a project, that linter check is not passing. Kindly fix that.
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **Does not count** into the code reviews limit._

THANKS for the response:
I put it in the comment that me and my coding partner working for quite a while to fix this issue and he got a response from the senior's developers that:

NOTE: **I have tried all possible ways to solve the linter error on GitHub actions but it's not working, however locally all the linters are passing. If you can assist in solving or give any idea it would be great. I asked the senior code reviewer and he told me to submit it for review. kindly review my Project. Thanks
**

The senior code reviewer told us to write this NOTE here.

I put the message in the dashboard as well. I think it has been missed.

Here is the link for the help and have not received any response:
https://questions.microverse.org/t/document-is-not-defined-resourceerror-document-is-not-defined/3020

Can you help me to figure it out why we are getting this issue?

**Comment by Mre55** [comment]
> > **Hi @gak92,**
> > While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> > ## Invalid Code Review Request ❎
> > You have submitted a project, that linter check is not passing. Kindly fix that.
> > Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> > Cheers and Happy coding!👏👏👏
> > Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> > **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> > _Invalid Code Review Request **Does not count** into the code reviews limit._
> 
> THANKS for the response: I put it in the comment that me and my coding partner working for quite a while to fix this issue and he got a response from the senior's developers that:
> 
> NOTE: **I have tried all possible ways to solve the linter error on GitHub actions but it's not working, however locally all the linters are passing. If you can assist in solving or give any idea it would be great. I asked the senior code reviewer and he told me to submit it for review. kindly review my Project. Thanks **
> 
> The senior code reviewer told us to write this NOTE here.
> 
> I put the message in the dashboard as well. I think it has been missed.
> 
> Here is the link for the help and have not received any response: https://questions.microverse.org/t/document-is-not-defined-resourceerror-document-is-not-defined/3020
> 
> Can you help me to figure it out why we are getting this issue?

@gak92 I'll get by to you shortly kindly wait for the response.


Hello, @gak92 I talked to seniors about that and try to implement what was answered in [here](https://questions.microverse.org/t/document-is-not-defined-resourceerror-document-is-not-defined/3020) about your question if it will not work you can submit again while having the linter error the next code reviewer will `APPROVE` it right away.

**Review by AmaduKamara — COMMENTED** [review]
Hi @gak92,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is having a Web hint error. Kindly fix all linter errors in your project and then ask for a review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
```

**Comment by gak92** [comment]
> Hi @gak92,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request
> You have submitted a project, that is having a Web hint error. Kindly fix all linter errors in your project and then ask for a review.
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

@AmaduKamara 
I think this is the error which many people had it.
Please check this link and read the comment from the last code reviewer:
https://questions.microverse.org/t/document-is-not-defined-resourceerror-document-is-not-defined/3020

Please check it:
"@gak92 I'll get by to you shortly kindly wait for the response.

Hello, @gak92 I talked to seniors about that and try to implement what was answered in [here](https://questions.microverse.org/t/document-is-not-defined-resourceerror-document-is-not-defined/3020) about your question if it will not work you can submit again while having the linter error the next code reviewer will APPROVE it right away."

My coding partner also had the same issue and we discuss with seniors developers and they mentioned that:

NOTE: **I have tried all possible ways to solve the linter error on GitHub actions but it's not working, however locally all the linters are passing. If you can assist in solving or give any idea it would be great. I asked the senior code reviewer and he told me to submit it for review. kindly review my Project. Thanks
**

The senior code reviewer told us to write this NOTE here.

My coding partner got review with this error.

Thanks

**Comment by gak92** [comment]
> My coding partner got review with this error.

@AmaduKamara 
Or I would be thankful to you if you can help me to sort out this error as I am struggling with this for the last two days and could not find any solution.
Thanks

File: README.md [hunk 1]
```diff
@@ -1,67 +1,67 @@
...
+## Authors
+
+👤 **Author1**
+
+- GitHub: Ghazanfar Ali [@gak92](https://github.com/gak92)
+- LinkedIn: Ghazanfar Ali (https://www.linkedin.com/in/ghazanfar-ali-9a4998a/)
+
+👤 **Author2**
+- GitHub: [@githubhandle](https://github.com/ogagaoghene)
+
```
Comment (codepantha) [inline README.md]: - [ ] Please include the details of your coding partner as Author 2 as this is a collaborative project 👍

File: index.html [hunk 2]
```diff
@@ -1,389 +1,386 @@
...
+          <a href="#">My Logo</a>
+        </div>
+
+        <nav class="navbar">
+          <img src="./images/Union.png" alt="Menu Icon" class="icon menu" />
+          <i class="bi bi-x"></i>
+          <ul class="navbar-list">
+            <li class="navbar-item">
+              <a href="#" tabindex="1">Portfolio</a>
+            </li>
```
Comment (codepantha) [inline index.html]: - [ ] When the portfolio link in the mobile menu is clicked, it doesn't take me to the correct part of the page. All other links are working except this one. Please modify it 🙏

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUESTED CHANGES ♻️ 

## Hi team 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### Highlights 🎯 
✔️ Descriptive PR
✔️ Meaningful commit messages
✔️ Passing linter checks

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nil

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @gak92     , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

[OPTIONAL] Kindly add a screenshot to your README file showing your webpage user Interface. You can use extentions like fireshot to capturer the entire page.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**